### PR TITLE
279 rebrand mailer

### LINKED ISSUE #279: [IMPROVEMENT] - Weekend-Fly rebranding mailer

### PR DESCRIPTION
**Pull request by christinasugiono**
image

closes #279

### CODE REVIEW

**Comment by christinasugiono** [comment]
image

File: app/views/layouts/default_wef.html.erb [hunk 1]
```diff
@@ -6,49 +6,128 @@
...
+      }
     </style>
   </head>
 
   <body>
-    <div class="container">
-    <%= image_tag(attachments["full-logo-early-dark.png"].url, alt: "Weekend-Fly logo", height: "64px") %>
-
-      <%= yield %>
+    <div class="background" style="background-image: url(<%= image_path(attachments["background.png"].url) %>);">
```
Comment (alexstan67) [inline app/views/layouts/default_wef.html.erb:124]: I didn't find the correct way to do, but for sure this won't work. Try something like:
````ruby
<div class="background" style="background-image: url("<%= attachments["background.png"].url %>");">
````

**Comment by alexstan67** [comment]
I came to a more basic layout respecting Maria's brandbook:
image

**Comment by alexstan67** [comment]
closes#279

**Pull request merged**